src/libraries/Native/Unix/System.Native/pal_networking.c

```diff
@@ -2422,9 +2426,45 @@ int32_t SystemNative_Socket(int32_t addressFamily, int32_t socketType, int32_t p
     return Error_SUCCESS;
 }
 
-int32_t SystemNative_GetSocketType(intptr_t socket, int32_t* addressFamily, int32_t* socketType, int32_t* protocolType)
+#if HAVE_SYS_PROCINFO_H
+int32_t SystemNative_GetSocketType(intptr_t socket, int32_t* addressFamily, int32_t* socketType, int32_t* protocolType, int32_t* isListening)
```

Comment:
`isBlocking` in C# signature, `isListening` here?

Reply:
It should be in Interop.GetSocketType.cs, unless I misunderstand your question.

Reply:
Looks like this was either fixed or I misread the code initially as I can't find it in the current commit. 👍

Reply:
yes, its as fixed as @stephentoub comment on it as well.